US County Data - pandas.ipynb

```diff
@@ -2,17 +2,20 @@
  "cells": [
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #6.    warnings.simplefilter(action='ignore', category=pd.errors.PerformanceWarning)
</pre><p>cheating :) I mean not literally cheating, but the warnings sometimes give us useful information, like "this is not a great way to do this if you care for performance"</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/CMU-MS-DAS-Modern-Programming-Mini/ClassAssignmentNo4/pull/3/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>